### ISSUE TITLE
Colorprofile is kept on WIN OS, but not on LINUX OS

### ISSUE THREAD

**Issue by CONTENTSERVSupport** [ImageMagick/ImageMagick#846]
On Windows operating system,

In Imagemagick when the image is rendered from cmyk to rgb the color profile is kept.

Like for example when the image "testimg.jpg" which is CMYK is rendered to image "finalimg.png" for RGB colorspace and having
colorprofile as "AppleRGB.icc" , by the command

convert testimg.jpg -colorspace rgb -profile AppleRGB.icc finalimg.png

the colorprofile "AppleRGB.icc" is kept in Windows operating system.

On Linux operating system,

In Imagemagick when the image is rendered from cmyk to rgb the color profile is not kept.

Like for example when the image "testimg1.jpg" which is CMYK is rendered to image "finalimg1.png" for RGB colorspace and having
colorprofile as "AppleRGB.icc" , by the command

convert testimg1.jpg -colorspace rgb -profile AppleRGB.icc finalimg1.png

the colorprofile "AppleRGB.icc" is not kept in Linux operating system.

So, the conclusion is in Windows Opearting system the colorprofile is kept but in Linux operating system there is problem in applying the colorprofile.

########

Question: Which behavior is correct? Needs a bugfix to be done by your dev team?

**Comment by dlemstra** [comment]
Could you not delete the issue template the next time? We cannot answer your question if we don't know which versions of ImageMagick you are using on both machines.

**Comment by dlemstra** [comment]
Could you give it another try with the latest release on both platforms?

**Comment by CONTENTSERVSupport** [comment]
Our developer tested with latest verison of Imagemagick

Version: ImageMagick 7.0.7-10 Q16 x86_64 2017-11-07

it's not working .

**Comment by CONTENTSERVSupport** [comment]
Since 2 weeks no updates.

Please be so kind and report any news on this - thank you!

**Comment by CONTENTSERVSupport** [comment]
Info from our developer:

Nope. Seems that behaviour is this way:
convert(== create new pic out an existing one) existing.pic sourceProfile tgtProfile master.png already does the conversion to targetcolorspace.
after that we tried to change the colorspace via mogrify (== work on the specified img directly) from sourceProfile to tgtProfile directly on the png.
since this is already a sRGB an error occurred and the tmpfile (with ~ at the end) stayed there and what was basically done was that for some kind of reason the cmyk sourceprofile got embedded within the png.
on win this is not occurring...
so basically this is still a IM bug, but it was solved by skipping the unnecessary conversion.
